interface/i18n/mist.es.i18n.json

```diff
@@ -0,0 +1,191 @@
+                "dropFilesHere" : "Soltar el archivo de la pre-venta",
+                "creating": "Creando...",
+                "importing": "Importando...",
+                "skip" : "Omitir este paso",
+                "next" : "Siguiente",
+                "protectTitle" : "Proteger su cuenta",
+                "protectDescription" : "Elija una contraseña para su nueva cuenta que sea tan fuerte como si fuera a proteger las llaves de su casa!",
+                "accountTitle" : "Cargarla!",
+                "accountTitleTestnet" : "Minar algunos!",
+                "etherDescription": "La red ethereum está basada en un token llamado “Ether”. Necesitarás una pequeña cantidad de él para hacer cualquier cosa en la red Ethereum.",
+                "loadItDescription" : "Si ya dispone de Bitcoin o cualquier otra criptomoneda, puede convertirlas fácilmente a ether utilizando shapeshift. <br><br> Recomendamos una cuantía de entre <strong>0.25 a 1 ether</strong>.",
+                "mineItDescription": "<strong>En la red de pruebas puedes minar ether tú mismo desde el menú <em>Desarrollar</em> y eligiendo <em>Empezar a minar</em>. </strong> <br> <strong> NO INTENTE ENVIAR ETHER REALES A ESTA DIRECCIÓN </strong>",
+                "you" : "TÚ",
+                "etherbase" : "Cuenta principal (etherbase)",
+                "depositBitcoin" : "Depositar Bitcoin",
+                "learnIt" : "Aprenda mientras espera",
+                "downloadingBlocks": "Descargando bloques",
+                "block": "Bloque",
+                "tutorial1Description" : "<p> Ahora lo único que queda por hacer es esperar a que finalice la descarga. Aquí tiene unas sugerencias de lectura: </p> <h4>Haga su propio dinero </h4> <p> Haga una criptomoneda con una oferta de mercado fija, tokens que representen activos del mundo real, etc </p>",
+                "tutorial2Description" : "<h4> Crear una crowdsale </h4> <p> Recaude fondos para un objetivo común, totalmente confiable y sin terceros. Evite así el obstáculo de la financiación del sistema tradicional y vaya directamente a la fuente mediante la financiación de una organización a través de la blockchain.</p>",
+                "tutorial3Description" : "<h4> Organización en la Blockchain </h4> <p> Crea una organización autónoma con reglas inquebrantables sobre cómo el dinero puede ser gastado y quién puede tomar las decisiones. Deje que sus promotores tengan un papel activo en la forma en que se gasta el dinero</p>",
+                "buttons": {
+                    "showPassword": "Mostrar contraseña",
+                    "importAccount": "Importar cuenta",
+                    "launchApp": "Lanzar Aplicación!",
+                    "learnReceipt": "Aprender esta receta"
```

Comment:
Thanks for your appreciation @TheBitcoin, nevertheless in order to refer to "receptor or recibidor" as you've said, term would be "receptor or receiver" and not the one that we have in the mist strings: recipe".

You can check out other translations like the french one 😉